stack.yaml

```diff
@@ -1,4 +1,4 @@
-resolver: lts-8.6
+resolver: lts-8.13
```

Comment:
You can probably also remove most of the `extra-deps` below, since `lts-8.13` contains `proto-lens-*` and `lens-labels`:
https://www.stackage.org/lts-8.13/package/proto-lens-0.2.0.1